cmake/FindBlosc.cmake

```diff
@@ -53,7 +65,7 @@ IF ( BLOSC_FOUND )
     CACHE STRING "Blosc library directories")
 
   SET ( _blosc_library_name "blosc" )
-  
+
```

Comment:
I'd rather the trailing space changes were in a separate PR.

Reply:
I'll see if I can break the whitespace changes out (my editor automatically removes them so it's not something I usually think about).